### PR TITLE
update readme.md for docker

### PR DESCRIPTION
**Pull request by viveksharmaui**
We previously discussed in PR #52  to add docker commands in readme.md file so i did that please review .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,14 +36,21 @@ To view a live example, **[click here](https://saadpasta.github.io/)**
...
 ```
 
+### Docker Commands 
+
+```
+1) BUILD IMAGE : docker build -t developerfolio:latest .
+2) RUN IMAGE: docker run -p 3000:3000 developerfolio:latest
+```
+
 ---
```
Comment (kartik918) [inline README.md]: @saadpasta do you think this breaker needs to exist since it looks pretty inconsistent with the rest of the documentation?

Comment (kartik918) [reply README.md]: Also move these changes in a single commit under your already existing PR if you can @viveksharmaui

Comment (viveksharmaui) [reply README.md]: @kartik918 sure!

Comment (saadpasta) [reply README.md]: @kartik918 What do you say ?

Comment (saadpasta) [reply README.md]: Should we move this at the end

Comment (kartik918) [reply README.md]: Yes, that would be better. @viveksharmaui move the hr breaker to the end and update docker instructions in readme in your existing "Dockerized" branch PR. I'll close this PR after that one gets merged.

**Comment by kartik918** [comment]
This PR has been moved under a commit in #52.

**Pull request closed without merging**